packages/datetime/src/_datepicker.scss

```diff
@@ -97,6 +97,14 @@ $header-margin: ($header-height - $pt-input-height) / 2 !default;
       color: $pt-text-color-disabled;
     }
 
+    &.DayPicker-Day--isToday {
+      span {
+        border: 1px solid $pt-divider-black;
+        border-radius: $pt-border-radius;
+        padding: 6px;
```

Comment:
from my CSS tweaking in the inspector, I think this should be 7px.

otherwise, you currently get this:

![image](https://user-images.githubusercontent.com/723999/60446545-96bd7b00-9bef-11e9-9c16-de7c9f944905.png)

Reply:
Done